src/cli/commands/common/prompt-for-aws-target.ts

```diff
@@ -126,3 +165,162 @@ function toClusterChoice(dbCluster: AwsDbCluster): DistinctChoice {
     },
   };
 }
+function toElasticacheClusterChoices(
```

Comment:
Please, extract all these elasticache-related functions to a separate file in the same folder called `to-elasticache-choices.ts`